### PR TITLE
Added order lines csv

### PR DESCRIPTION
**Pull request by acaldas**

### CODE REVIEW

File: src/budy/controllers/api/order.py [hunk 1]
```diff
@@ -401,3 +401,35 @@ def cancel(self, key):
...
+            "product",
+            "quantity",
+            "size",
+            "scale",
+            "total",
+            "currency",
+            "country"
+        )]
+        for line in order.lines:
+            if not line.product: continue
```
Comment (tsilva) [inline src/budy/controllers/api/order.py]: @joamag not sure if lines without products should be ignored or included

Comment (joamag) [inline src/budy/controllers/api/order.py]: @tsilva products should be considered mandatory for a line

Comment (joamag) [inline src/budy/controllers/api/order.py]: @tsilva include also `line.product.product_id`

**Comment by joamag** [comment]
@acaldas also include information about the order (eg: order id, date, account)

**Pull request merged**